maro/rl/rollout/env_sampler.py

```diff
@@ -236,29 +240,27 @@ def __init__(
 
     @abstractmethod
     def _get_global_and_agent_state(
-        self, event: object, tick: int = None
+        self, event: object, tick: int = None,
     ) -> Tuple[Optional[np.ndarray], Dict[Any, np.ndarray]]:
-        """
-        Get global state and dict of agent states.
+        """Get global state and dict of agent states.
 
         Args:
             event (object): Event.
-            tick (int): Tick.
+            tick (int, default=None): Current tick.
 
         Returns:
-            Global state: np.ndarray
-            Dict of agent states: Dict[Any, np.ndarray]
+            Global state (np.ndarray)
+            Dict of agent states (Dict[Any, np.ndarray])
         """
         raise NotImplementedError
 
     @abstractmethod
     def _translate_to_env_action(self, action_dict: Dict[Any, np.ndarray], event: object) -> Dict[Any, object]:
-        """
-        Translation the actions into the format that the env could recognize.
+        """Translation the actions into the format that the env could recognize.
```

Comment:
Translate model-generated actions into an object that can be executed by the env.